public/stylesheets/question.css

```diff
@@ -0,0 +1,113 @@
+.prompt {
+    /* font-family: Poppins; */
+    font-style: normal;
+    font-weight: bold;
+    font-size: 30px;
+    color: #1652A3;
+}
+
+
+.jumbotron {
+    background-color: #1377FF;
+}
+
+.jumbotron h1 {
+    color: white;
+    font-weight: bolder;
+}
+
+
+ul li {
+    list-style: none;
+    font-size: 20px;
+}
+
+.answer {
+    font-size: 20px;
+}
+
+
+
+/* 
+    git commit (other branch)
+    git push (other branch)
+    git pull (on master)
+    git merge master (other branch)
+    write message 
+    :wq save and quit
+*/
+
+
+
+
+
+.step_progressbar {
+    display: block;
+    width: 100%;
+    /* margin: 100px auto; */
+    /* background-color: #ffff; */
+}
+
+.progress-list {
+    display: block;
+    list-style: none;
+    align-items: center;
+    justify-content: space-between;
+    counter-reset: step;
+}
+
+
+.progress-item {
```

Comment:
Nice job on the styling.

Here's something you might find an interesting read on how to name/structure css classes to make things easier to understand and read. I think you've already implicitly done some of it, as it is really just a common sense set of rules:

http://getbem.com/naming/